docs/core/diagnostics/snippets/Microsoft.Diagnostics.NETCore.Client/csharp/PrintRuntimeGCEvents.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.Diagnostics.NETCore.Client;
+using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.EventPipe;
+using Microsoft.Diagnostics.Tracing.Parsers;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+
+public class RuntimeGCEventsPrinter
+{
+    public static void PrintRuntimeGCEvents(int processId)
+    {
+        List<EventPipeProvider> providers = new List<EventPipeProvider>()
```

Comment:
```suggestion
        var providers = new List<EventPipeProvider>()
```

Use `var` where the type is obvious.